### PR TITLE
libnghttp2 nghttp2 1.54.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/libnghttp2.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Libnghttp2 < Formula
   desc "HTTP/2 C Library"
   homepage "https://nghttp2.org/"
-  url "https://github.com/nghttp2/nghttp2/releases/download/v1.53.0/nghttp2-1.53.0.tar.gz"
-  mirror "http://fresh-center.net/linux/www/nghttp2-1.53.0.tar.gz"
-  mirror "http://fresh-center.net/linux/www/legacy/nghttp2-1.53.0.tar.gz"
```
Comment (chenrui333) [inline Formula/libnghttp2.rb:6]: removing this (feel this is a bit redundant)

Comment (Bo98) [reply Formula/libnghttp2.rb:6]: It's not redundant - note how `http://fresh-center.net/linux/www/nghttp2-1.53.0.tar.gz` no longer exists but `http://fresh-center.net/linux/www/legacy/nghttp2-1.53.0.tar.gz` does.

Comment (Bo98) [reply Formula/libnghttp2.rb:6]: Well right now before this PR is merged is a perfect example of when installing `libnghttp2` would be broken if legacy wasn't there.

Comment (chenrui333) [reply Formula/libnghttp2.rb:6]: yeah, that is true 😆

**Review by SMillerDev — APPROVED** [review]

**Pull request merged**